person.rb

```diff
@@ -20,6 +22,10 @@ def correct_name
     @name
   end
 
+  def add_rental(person, date)
+    Rental.new(date, self, person)
```

Comment:
Same here, the association should be set up by passing the `book` instead of `person`. The `self` here will refer to the `person`. Kindly fix this mistake.

Reply:
Yes this is correct @derrick1451 I made a typo in my comment

Reply:
thanks @Munsa1